tooling/charts/do500/values.yaml

```diff
@@ -66,3 +67,17 @@ crw:
 #    source_repo: "https://github.com/rht-labs/enablement-docs"
 #    s2i_name: "nodejs"
 #    s2i_tag: "10"
+
+sealed-secrets:
+  # Disabled by default
+  enabled: true
+  nameOverride: sealed-secrets
+  fullnameOverride: sealed-secrets
+  # namespace must exist, so we use labs-ci-cd by default.
```

Comment:
is `labs-ci-cd` a carry over from old content?

Reply:
Yup - i removed the comment